packages/client/src/components/pages/muster-page/table-unit-muster-compliance.tsx

```diff
@@ -0,0 +1,227 @@
+import {
+  Grid,
+  Paper,
+  Table,
+  TableBody,
+  TableContainer,
+  TableCell,
+  TableHead,
+  TableRow,
+} from '@material-ui/core';
+
+import * as Plotly from 'plotly.js';
+import React, {
+  useCallback,
+  useEffect,
+  useState,
+} from 'react';
+import Plot from 'react-plotly.js';
+
+import { UnitSelector } from '../../../selectors/unit.selector';
+import { Modal } from '../../../actions/modal.actions';
+import { formatErrorMessage } from '../../../utility/errors';
+import { UserSelector } from '../../../selectors/user.selector';
+import { MusterClient } from '../../../client/muster.client';
+import { useAppDispatch } from '../../../hooks/use-app-dispatch';
+import { useAppSelector } from '../../../hooks/use-app-selector';
+
+export const UnitMusterComplianceTable = (props: any) => {
+  const dispatch = useAppDispatch();
+  const units = useAppSelector(UnitSelector.all);
+  const trendChart = {
```

Comment:
`trendChart` is immediately consumed on line 62 but that is not evident unless one search for uses. I would get rid of it and construct the object on the line 62.  You could even move all this code to a separate module if you like.